src/components/Episodes/renderUtils.jsx

```diff
@@ -1,29 +1,36 @@
 import React from "react";
 import Episode from "COMPONENTS/Episode/";
+import { TrailersProvider } from "./";
+import UTILS from "SHARED/utils.js";
 
-function renderEpisode(data, index){
+function renderEpisode(data, index, prevTrailer, nextTrailer){
 
 	const { 
 		number,
 		title,
 		synopsis,
-		image,
-		video,
+		thumbnail,
+		trailer,
 	} = data;
 
-	const id = `episode__${number}`;
+	const id = UTILS.convertToSafeString(title, "-");
 
 	return (
-		<Episode
-			id={ id }
-			key={ id }
-			index={ index }
-			number={ number }
-			title={ title }
-			synopsis={ synopsis }
-			image={ image }
-			video={ video }
-		/>
+		<TrailersProvider 
+			value={ { prevTrailer, nextTrailer } }
```

Comment:
could we have `next` and `previous` as the props instead of `value`?

Reply:
and rather than them being passed down as arguments in this `renderEpisode` function, it should just be:

```

const { trailer: nextTrailer } = data[index+1] || {};
const { trailer: prevTrailer } = data[index-1] || {};

<TrailerProvider
    next={nextTrailer}
    previous={prevTrailer}
>
    <Episode />
</TrailerProvider>
```